<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTechnic</class>
 <widget class="QWidget" name="AddTechnic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="add_technic_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить приём</string>
   </property>
  </widget>
  <widget class="QLabel" name="technic_name_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Название приёма</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="technic_name_text_edit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableView" name="techincs_table">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>521</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked</set>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="delete_technic_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить приём</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
